### PR TITLE
Autodetect llvm-config for QEMU bindings generation

### PR DESCRIPTION
**Pull request by andreafioraldi**

### CODE REVIEW

File: libafl_qemu/libafl_qemu_build/src/lib.rs [hunk 1]
```diff
@@ -25,6 +30,73 @@ pub fn build_with_bindings(
...
+    match Command::new(llvm_config).args(args).output() {
+        Ok(output) => String::from_utf8(output.stdout)
+            .expect("Unexpected llvm-config output")
+            .trim()
+            .to_string(),
+        Err(e) => panic!("Could not execute llvm-config: {e}"),
+    }
+}
+
+fn find_llvm_version(llvm_config: String) -> Option<i32> {
```
Comment (domenukk) [inline libafl_qemu/libafl_qemu_build/src/lib.rs:68]: Could we do that in targets or somewhere centralized? I think we need it in multiple places, right?

Comment (andreafioraldi) [reply libafl_qemu/libafl_qemu_build/src/lib.rs:68]: It is similar to the code in libafl cc but different

Comment (andreafioraldi) [reply libafl_qemu/libafl_qemu_build/src/lib.rs:68]: also, this thing runs always in build.rs, we may want a centralized build.rs lib maybe but not in libafl_targets for sure

**Comment by andreafioraldi** [comment]
CI failing because llvm is too old

**Pull request merged**